### ISSUE TITLE
cmd-/ key binding does not work

### ISSUE THREAD

**Issue by adrianfon9** [monome/maiden#95]
key binding in default (ace) does not function as comment toggle in the default (ace) editor even with focus within editing window.

**Comment by pq** [comment]
added context from [lines](https://llllllll.co/t/norns-maiden/14052/5):

> I am using safari 11.1.1 on 10.13.5

**Comment by pq** [comment]
hiya @dansimco .  i haven't cycled back to this but would be pretty surprised if it weren't still in an issue.  browser quirks. 😬 

or wait, are you on os x?  and using safari?  what versions?

what kinds of refinements did you have in mind?

**Comment by synthetiv** [comment]
I'm on Mac OS Mojave version 10.14.6, Safari 12.1.2, and Cmd-/ works fine for me when editing Lua files. It does NOT work on SuperCollider files, in either Safari, Chrome, or Firefox, but I'm sure that's because Ace doesn't know enough about SC to know what a line comment would look like, and I know there's another issue here for that.

If this is still an issue for the OP, it might be a conflict with a browser extension.

**Comment by dansimco** [comment]
As a workaround to a few quirks like this, I can recommend using https://fluidapp.com/ on Mac to make yourself a "Maiden" app. For one it saves me from accidentally backing-out of the page when pressing cmd+[ . Maiden also looks really nice in it's own app.

**Comment by synthetiv** [comment]
Good point! [Nativefier](https://www.npmjs.com/package/nativefier) is a solid open source option for this. More fiddly to configure, but you can even set up an app to be frameless (no titlebar). I haven't been using maiden long enough to do this yet, but I'm sure I will eventually...
